src/test/java/ai/promoted/delivery/client/SdkDeliveryTest.java

```diff
@@ -99,35 +95,6 @@ void testRetrievalInsertionOffsetWithOffsetOutsideSize() throws DeliveryExceptio
     assertEquals(0, resp.getInsertion().size());
   }
 
-  @Test
-  void testResponseInsertionsOnlyHaveKeyFields() throws DeliveryException {
```

Comment:
This test doesn't really make sense with the Protobuf class since the fields aren't optional. The only assertion that will compile is the content ID check, and the other tests already do it